source/request.js

```diff
@@ -146,7 +145,7 @@ Request.prototype.servedByRailgun = function () {
 };
 
 Request.prototype.servedOverH2 = function () {
-  return this.SPDY && this.connectionType === 'h2';
+  return this.connectionType === 'h2';
```

Comment:
The [RFC 7301 specification](https://tools.ietf.org/html/rfc7301#section-6) uses `http/2` for this.

Reply:
It does not, it only defines "http/1.1", "spdy/1", "spdy/2", and "spdy/3". In any case, [IANA](https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids) is canonical.